compare_vcf.py

```diff
@@ -287,10 +287,105 @@ def process(args):
+                content.append(line_strip)
+
+            else:
+                if content[-1][0] != "#":
+                    del content[-1]
+                content.append(line_strip)
+
+                single_var_file = utils.write_vcf(content, os.path.join(out_dir, "single.vcf"))
+                single_var_file = utils.sort_and_compress(single_var_file)
+
+                for i, item in enumerate(files_to_pair_with_clean):
+
+                    equivalent_variant = None
+
+                    if item:
+                        vcfeval_prefix = os.path.join(out_dir, 'vcfeval_compare_results_annotate')
+
+                        vcfeval_comparator = RTGVCFComparator(prefix=vcfeval_prefix, true_vcf = item, reference = sdf,
+                                         regions = None,
+                                         sample = sample, vcfs = [single_var_file],
+                                         exclude_filtered = False,
+                                         match_geno = False,
+                                         log_to_file= log_to_file,
+                                         opts = vcfeval_options)
+
+                        equivalent_variant = utils.get_equivalent_variant(line_split, vcfeval_comparator.get_tp())
+
+                        #clean up
+                        if os.path.exists(vcfeval_prefix):
+                            LOGGER.warn('{0} exists, removing ...'.format(vcfeval_prefix))
+                            shutil.rmtree(vcfeval_prefix)
+
+                    if i == 0:
+                        if equivalent_variant:
+                            try:
+                                AO = int(equivalent_variant[-1].split(':')[4].split(',')[0])
+                                RO = int(equivalent_variant[-1].split(':')[2].split(',')[0])
+                            except:
+                                info = "N/A;"
+                            else:
+                                info = str(float(AO)/(AO+RO)) + ';'
+                        else:
+                            info = "N/A;"
+
+                    elif i == 1:
+                        if equivalent_variant:
+                            info += equivalent_variant[0]+'_'+equivalent_variant[1]+'_'+equivalent_variant[3]+'_'+equivalent_variant[4]+'_'+equivalent_variant[-1] + ";"
+                        else:
+                            info += "N/A;"
+
+                    elif i == 2:
+                        info += "pure;" if not equivalent_variant else "not;"
+
+                line_split[6] = info
+                annotated_content.append('\t'.join(line_split))
+
+                #clean up
+                if os.path.isfile(single_var_file):
+                    os.remove(single_var_file)
+                    os.remove(single_var_file+".tbi")
+
+    annotated_file = utils.write_vcf(annotated_content, os.path.join(args.out_dir, "{}_annotated.vcf".format(os.path.splitext(os.path.splitext(os.path.basename(augmented_file))[0])[0])))
```

Comment:
Why is `splitext` used twice here? Wonder if direct string removal (say `rstrip`) would be simpler?

Reply:
Good spot, I'll change this